ESAnalysis/init_dos.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python3
+
+"""
+Script that uses WaNo input to write QuantumPatch input.
+"""
+
+import yaml, copy
+from yaml import CLoader
+import sys
+import math
+
+from QuantumPatch.Shredder.MoleculeSystem import MoleculeSystem
+from QuantumPatch.Shredder.Parsers.ParserCommon import parse_system
+
+class QuantumPatchWaNoError(Exception):
+    pass
+
+
+
+def get_disorder_settings(wano, cpu_per_node=1):
+    # Shorthands
+    wano_general = wano["Tabs"]["General"]["General Settings"]
+    wano_core = wano["Tabs"]["General"]["Core Shell"]
+    wano_env = wano["Tabs"]["General"]["Shell for Disorder and Couplings"]
+    wano_engines = wano["Tabs"]["Engines"]
+    wano_storage = wano["Tabs"]["Storage"]
+
+    with open("disorder_settings_template.yml", "r") as qpngin:
+        cfg = yaml.load(qpngin, Loader=CLoader)  # Script will modify this and re-write it
+
+    # Shells
+    number_core_mols = min(100, wano_env["Number of molecules"])
```

Comment:
I would make sure this is an int. Cast to int

Reply:
done